packages/mui-material/src/utils/getChildRef.js

```diff
@@ -0,0 +1,6 @@
+export default function getChildRef(children) {
```

Comment:
Can we use TypeScript to define this util? Or at least add the corresponding .d.ts file